packages/app-store/googlecalendar/lib/CalendarService.test.ts

```diff
@@ -1,12 +1,11 @@
-import prismaMock from "../../../../tests/libs/__mocks__/prismaMock";
+import prismock from "../../../../tests/libs/__mocks__/prisma";
```

Comment:
Slight typo
```suggestion
import prisamock from "../../../../tests/libs/__mocks__/prisma";
```

Reply:
That's actually what I need. `prismock` is a library that provides in Memory DB, no mocking required.

and `prismaMock` is a mock created by us that simply does a deep mock for every method of prisma without any implementation of these methods. We then mock as required.